pyproject.toml

```diff
@@ -134,7 +133,8 @@ full = [
     "cartopy>=0.22.0",
     "firefly>=3.2.0",
     "glueviz>=0.13.3",
-    "ipython>=2.0.0",
+    "ipython>=7.16.2",
```

Comment:
While I was at it, I bumped minimal requirement for IPython here to the oldest version that supported Python 3.9 (i.e. the first one that was released *after* Python 3.9.0)

Reply:
haha, I love that we had it at 2.0 for a bajillion years. :)